GitUI/FontUtil.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace GitUI
+{
+    using System;
+    using System.Drawing;
+
+    public static class FontUtil
+    {
+        static FontUtil()
+        {
+            var hTheme = NativeMethods.OpenThemeData(IntPtr.Zero, "TEXTSTYLE");
+
+            var success = false;
+
+            if (hTheme != IntPtr.Zero)
+            {
+                // TODO: use C# 7.0 out var parameters
+                NativeMethods.LOGFONT pFont;
+                success = NativeMethods.GetThemeFont(hTheme, IntPtr.Zero, NativeMethods.TEXT_MAININSTRUCTION, 0, NativeMethods.TMT_FONT, out pFont) == NativeConstants.S_OK;
```

Comment:
Interesting, how could/would the compiler convert the `int32` returned by `GetThemeFont` to `bool`? Shouldn't we compare it to the `S_OK`?

Reply:
That's what it's doing (maybe I should have put `NativeConstants.S_OK ==` first :)

Reply:
The line is too long, didnt see the rest.

Reply:
I split this out onto 2 lines to make it clearer.